src/android/device.rs

```diff
@@ -176,6 +397,83 @@ impl<'a> Device<'a> {
         Ok(())
     }
 
+    fn clean_apks(&self, config: &Config, profile: Profile) -> Result<(), ApksBuildError> {
+        let flavor = self.target.arch;
+        let apks_path = Self::apks_path(config, profile, flavor);
+        bossy::Command::impure_parse("rm -f")
```

Comment:
why does this need to be forced + can you use `std::fs::remove_file` here, so it's a bit less scary?

Reply:
I think I did the forcing just because it would failed if it didn't already exist otherwise. But going with fs definitely feels better.